query-node/mappings/src/membership.ts

```diff
@@ -651,46 +653,62 @@ async function processMemberRemark(
   store: DatabaseManager,
   event: SubstrateEvent,
   memberId: MemberId,
-  decodedMessage: MemberRemarked
+  decodedMetadata: DecodedMetadataObject<IMemberRemarked> | null
 ): Promise<Partial<MetaprotocolTransactionSuccessful>> {
-  const messageType = decodedMessage.memberRemarked
+  if (decodedMetadata?.createApp) {
+    await processCreateAppMessage(store, event, decodedMetadata.createApp!, memberId.toString())
```

Comment:
Inside this if-block the non-null assertion is no longer needed:
```suggestion
    await processCreateAppMessage(store, event, decodedMetadata.createApp, memberId.toString())
```